riscv-asm.md

```diff
@@ -525,6 +528,82 @@ Integers in the subinterval [-2048, -1] can also be passed by their (unsigned) a
 in the interval [0xfffff800, 0xffffffff] on RV32I, and
 in [0xfffffffffffff800, 0xffffffffffffffff] on both RV32I and RV64I.
 
+Load Floating Point Immediate
+-------------------
+
+`Zfa` extension has provide `fli.{h|s|d|q}` instruction for loading a specific
```

Comment:
Does this need to be Zfa-specific?

We could instead define a generic way of supported floating-point literal formats supported by the assembler.
Example uses of such floating-point literals would be in `.single <expr>` and `fli.s fN, <expr>`.

The restriction of supported values for `fli` is a restriction that is instruction specific and does not need to be documented here.

Reply:
Yeah, I thought same thing, that will require one extra temporal register to hold the address, but that should not really big deal.